tests/pulseaudio_dbus_spec.lua

```diff
@@ -10,7 +10,7 @@ b.describe("PulseAudio with DBus", function ()
            b.before_each(function ()
                local address = pulse.get_address()
                local connection = pulse.get_connection(address)
-               local core = pulse.get_core(connection)
+               core = pulse.get_core(connection)
```

Comment:
This change is making `core` global. It may be worth testing `core` in its own `describe` block.

Reply:
I have just realised that this change appears in another commit, but the comment still applies ;)